qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -351,6 +374,159 @@ def target(self) -> Target:
     def meas_map(self) -> List[List[int]]:
         return self.options.meas_map
 
+    def configuration(self) -> PulseBackendConfiguration:
```

Comment:
What is the main reason for adding the V1 fields here? In terra its for backwards compatibility, but since this is new we could have a clean break (unless there is other code that depends on these this simulator is intended for use with, and that can't be updated).